go.mod

```diff
@@ -367,6 +367,7 @@ require (
 	github.com/go-ozzo/ozzo-validation v3.6.0+incompatible // indirect
 	github.com/goccy/go-json v0.10.2 // indirect
 	github.com/godbus/dbus v0.0.0-20190726142602-4481cbc300e2 // indirect
+	github.com/gofrs/flock v0.8.1 // indirect
```

Comment:
This introduced dependency is incompatible with the solaris build. As a result, we've downgraded the OCI sdk to a version that doesn't rely on `flock`.